lib/Sema/SemaExpr.cpp

```diff
@@ -12359,6 +12359,50 @@ ExprResult Sema::ActOnRangeBoundsExpr(SourceLocation BoundsKWLoc,
                                        RParenLoc);
 }
 
+ExprResult Sema::CreateRangeBoundsExpr(SourceLocation BoundsKWLoc,
```

Comment:
Please rename this to ActOnRangeBoundsExpr.   Create* functions are used to allocate data and not do any semantic checking.